src/common/timer.jl

```diff
@@ -0,0 +1,6 @@
+const to = TimerOutput()
+# creating a new macro to shorten the call :
+# replaces `@timeit to [label] [block]` by `@timer [label] [block]`
+macro timer(args...)
+    TimerOutputs.timer_expr(__module__, false, to, args...)
+end
```

Comment:
I'm starting to think whether we should call this `@timed` instead. `@timed function` reads nicer than `@timer function`.

Reply:
I don't know, on the one hand it's true that it reads nicer but on the other hand, it's an object that measures time, so calling it `timer` makes sense :thinking: .

Reply:
Actually the thing which measures time is `to`. So from that point of view I suggest to rename `to -> timer`, `timer -> timed`.

Reply:
Sounds good to me !

Reply:
But then we would overwrite the original `Main.@timed`macro also.

Reply:
Ah I forgot about that ... probably that we should not do. `@timing` maybe?

Reply:
Yeah, good for me.